lib/ProductOpener/Users.pm

```diff
@@ -348,6 +348,9 @@ sub check_user_form($$$) {
 	if (not $address) {
 		push @{$errors_ref}, $Lang{error_invalid_email}{$lang};
 	}
+	
+	# If all checks passed, reinitialize with modified email
+	$user_ref->{email} = Email::Valid->address($user_ref->{email});
```

Comment:
I think it would be better to do that in a else clause and use the $address which already has the result of Email::Valid->address( -address => $user_ref->{email}, -mxcheck => 1 );

Reply:
@stephanegigandet The $address variable becomes a boolean, if a "$@" condition is satisfied. So, I didn't want to disturb that check. Instead, I reinitalized the variable separately.

Reply:
@stephanegigandet I just realized that now. Really sorry for not noticing it earlier! 
Did a new commit with the else clause and reinitalization with $address variable.

Reply:
Perfect, thanks!